src/pages/[platform]/build-a-backend/server-side-rendering/index.mdx

```diff
@@ -211,7 +211,11 @@ In this example, if the incoming request is not associated with a valid user ses
 
 <Callout>
 
-**NOTE:** When calling `fetchAuthSession` with a `response` context, it will send the refreshed tokens (if any) back to the client via the `Set-Cookie` header in the response.
+When calling `fetchAuthSession` with a `response` context, it will send the refreshed tokens (if any) back to the client via the `Set-Cookie` header in the response.
+
+**Note:** The Next.js adapter currently doesn't support customization of cookie attributes from the server side, and cookies are always set with [default attribute values](https://github.com/aws-amplify/amplify-js/blob/2447917fb47ff616aca6f5d64f9c635cf738c7e3/packages/aws-amplify/src/adapter-core/storageFactories/createKeyValueStorageFromCookieStorageAdapter.ts#L10-L15).
+
+If you are changing the cookie store on the client side, you may need to manually set the cookie attributes to match the server-side configuration. If you need to customize the cookie attributes, you can use the lower level [`runWithAmplifyServerContext` function](/gen1/[platform]/build-a-backend/server-side-rendering/) instead.
```

Comment:
Should we remove the "gen1" in the linked SSR docs here?

Reply:
Good Q, we currently don't have a equivalent Gen2 page for the lower level adapter function. Open for suggestions.

Reply:
does this page have the info we're looking for? https://docs.amplify.aws/react/build-a-backend/server-side-rendering/#configure-amplify-apis-for-server-side-usage

Reply:
@josefaidt the Gen2 doc page contains only the information about the prebuilt Next.js adapter.

Reply:
If its in gen1 docs and gen2 but important to customers. Shouldnt we move this over?